### PR TITLE
Issue fixes

### PR DESCRIPTION
**Pull request by sarithapillai8**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "@date-io/date-fns": "^1.3.13",
     "@koale/useworker": "^3.4.0",
     "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
```
Comment (avaidyam) [inline package.json:34]: Can you please revert this change? We have already discussed multiple times to use `<Icon>icon_name_here</Icon>` instead of importing specific icons from this package. [If you need to find a specific icon, use the MaterialUI Icon Font page here.](https://fonts.google.com/icons?selected=Material+Icons)

File: src/components/NavigationLayout.tsx [hunk 2]
```diff
@@ -32,16 +32,13 @@ import {
...
 import { CredentialManager } from "./CredentialManager"
 import { ResponsiveMargin } from "./Utils"
-import classnames from "classnames"
 import ResponsiveDialog from "./ResponsiveDialog"
 import Messages from "./Messages"
 import LAMP from "lamp-core"
-import useInterval from "./useInterval"
 import { useTranslation } from "react-i18next"
-
+import AccountCircleOutlinedIcon from "@material-ui/icons/AccountCircleOutlined"
```
Comment (avaidyam) [inline src/components/NavigationLayout.tsx:39]: Please undo this change and use the appropriate `<Icon>` name.

File: src/components/Researcher/ParticipantList/PatientStudyCreator.tsx [hunk 3]
```diff
@@ -290,6 +300,40 @@ export default function PatientStudyCreator({
...
+                  <QRCode size={256} level="H" value={_qrLink(`${id}@lamp.com`, id)} />
+                </Grid>
+              </Tooltip>
+            </Grid>
+          </SnackMessage>
+        ),
+      }
+    )
+  }
+
```
Comment (avaidyam) [inline src/components/Researcher/ParticipantList/PatientStudyCreator.tsx:336]: [Why is this code duplicated from `AddUser.tsx`?](https://github.com/BIDMCDigitalPsychiatry/LAMP-dashboard/blob/007020c0bfa1423776f18ab67ac9403b014b0943/src/components/Researcher/ParticipantList/AddUser.tsx#L131) Code that is re-used in more than one place should always be refactored into a separate file and imported. Please change this.

File: package.json (hunk 1 above)
Comment (sarithapillai8) [reply package.json:34]: @avaidyam We tried to use the Icon. But it is not available there. I tried using that, the same icon was not available. That is why I used this.

Comment (sarithapillai8) [reply package.json:34]: If I can use this code - <span class="material-icons-outlined">account_circle</span>, I can remove that now. outlined icons are not  rendered. I tried <Icon>account_circle_outlined</Icon>

Comment (avaidyam) [reply package.json:34]: @sarithapillai8 [It looks like it is there though?](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aaccount_circle)

Comment (avaidyam) [reply package.json:34]: @sarithapillai8 That should be fine, to just use `account_circle` instead.

File: src/components/Researcher/ParticipantList/PatientStudyCreator.tsx (hunk 3 above)
Comment (sarithapillai8) [reply src/components/Researcher/ParticipantList/PatientStudyCreator.tsx:336]: @avaidyam As this is enqueueSnackbar, I was not able to make it as separate component. I tried to split the SnackMessage part from different component , but it was showing error " Cannot read property 'getBoundingClientRect' of null".

Comment (avaidyam) [reply src/components/Researcher/ParticipantList/PatientStudyCreator.tsx:336]: You should be able to make it a separate function (not a React component) in a separate file though?

File: src/components/Researcher/ParticipantList/NewPatientDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+              <Grid container justify="center" style={{ padding: 16 }}>
+                <QRCode size={256} level="H" value={_qrLink(`${id}@lamp.com`, id)} />
+              </Grid>
+            </Tooltip>
+          </Grid>
+        </SnackMessage>
+      ),
+    }
+  )
+}
```
Comment (avaidyam) [inline src/components/Researcher/ParticipantList/NewPatientDetail.tsx:47]: Please make sure the other usage of this (in AddUser.tsx) is also deleted and converted to calling this function.

Comment (sarithapillai8) [reply src/components/Researcher/ParticipantList/NewPatientDetail.tsx:47]: @avaidyam It is not working as expected. As I said above, it is showing some error. I can work on this on Monday.

**Comment by sarithapillai8** [comment]
@avaidyam  Issues specified are fixed. Please review and merge.
